src/components/Projects/Projects.tsx

```
@@ -0,0 +1,20 @@
+"use client";
+import { ProjectsData } from "@/db/main";
+import React from "react";
+import styles from "../../styles/components/Projects.module.scss";
+import Project from "./Project";
+type ProjectsProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)